python/doc/examples/probabilistic_modeling/generate_by_inversion.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Generate random variates by inverting the CDF\n",
+    "\n",
+    "## Abstract\n",
+    "\n",
+    "In this example, we present how to generate random variates by inversion of the cumulated distribution function. In simplest situations, this method is rarely used in practice, both for reasons of performance, statistical quality of the generated random variates and numerical accuracy of the generated numbers when we use floating point numbers. However, it is an interesting method to know about because it is a building block for other algorithms and can be used to visualize the distibution of the generated numbers. "
```

Comment:
```suggestion
    "In this example, we show how to generate random variates by inversion of the cumulated distribution function. In simple situations, this method is rarely used in practice because of problems with performance, statistical quality of the generated random variates and numerical accuracy of the generated numbers when we use floating point numbers. However, it is an interesting method to know about because it is a building block for other algorithms and can be used to visualize the distribution of the generated numbers. "
```